ruby_on_rails/app_initialisation.md

```diff
@@ -0,0 +1,86 @@
+# Initialise the Rails App
+
+* [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not. You can do this with `gem update rails`.
+
+* Start a new Rails project using the usual `rails new -d postgresql [project-name]` where the `project-name` is exactly the one you chose before.
+You may want to choose a different database from Postgres, but most of the time that will be your choice.
+If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
```

Comment:
very good thought! IMO would be also a part, i would mention in the very beginning of this chapter.